device utun0 : operation not supported

pod一直报错：
test/kubevpn-traffic-manager-57c4cd7b96-xrp4w[vpn]: 2024-12-05 10:48:25 tun.go:45 error: Create tun device error: can not setup IPv6 address efff:ffff:ffff:ffff:ffff:ffff:ffff:9999 to device utun0 : operation not supported
test/kubevpn-traffic-manager-57c4cd7b96-xrp4w[vpn]: 2024-12-05 10:48:25 route.go:90 error: Failed to create tun listener: can not setup IPv6 address efff:ffff:ffff:ffff:ffff:ffff:ffff:9999 to device utun0 : operation not supported
test/kubevpn-traffic-manager-57c4cd7b96-xrp4w[vpn]: 2024-12-05 10:48:25 serve.go:52 error: Parse server failed: can not setup IPv6 address efff:ffff:ffff:ffff:ffff:ffff:ffff:9999 to device utun0 : operation not supported
test/kubevpn-traffic-manager-57c4cd7b96-xrp4w[vpn]: Error: can not setup IPv6 address efff:ffff:ffff:ffff:ffff:ffff:ffff:9999 to device utun0 : operation not supported

--- a/pkg/tun/tun_linux.go
+++ b/pkg/tun/tun_linux.go
@@ -79,7 +79,7 @@ func createTun(cfg Config) (conn net.Conn, itf *net.Interface, err error) {
 		if ipv6, ipv6CIDR, err = net.ParseCIDR(cfg.Addr6); err != nil {
 			return
 		}
-		if err = netlink.NetworkLinkAddIp(ifc, ipv6, ipv6CIDR); err != nil {
+		if err = netlink.NetworkLinkAddIp(ifc, ipv6, ipv6CIDR); err != nil && !errors.Is(err, syscall.ENOTSUP) {
 			err = fmt.Errorf("can not setup IPv6 address %s to device %s : %v", ipv6.String(), name, err)
 			return
 		}

